### PR TITLE
Issue 531

### PR DESCRIPTION
**Pull request by yuktakul04**
## Description (e.g. "Related to ...", etc.)

_Please replace this description with a concise description of this Pull Request._

## Code review checklist (for code reviewer to complete)

- [ ] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR)
- [ ] Title summarizes what is changing
- [ ] Commit messages are meaningful (see [this][commit messages] for details)
- [ ] Tests have been included and/or updated, as appropriate
- [ ] Docstrings have been included and/or updated, as appropriate
- [ ] Changelog has been updated, as needed (see [CHANGELOG.md][changelog])

[changelog]: https://github.com/openlawlibrary/taf/blob/master/CHANGELOG.md
[commit messages]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: tests/issue-531.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import sys
+
+def cli_update_repo():
```
Comment (renatav) [inline tests/issue-531.py:3]: This shouldn't be a test, it needs to be integrated into the cli, so `taf\tools\repo\__init__.py`

**Review by renatav — COMMENTED** [review]
This should

File: tests/issue-531.py (hunk 1 above)
Comment (yuktakul04) [reply tests/issue-531.py:6]: I edited the code to import click according to __init__.py file.

**Comment by renatav** [comment]
I implemented this since I want to include this in the upcoming release. This is the commit: https://github.com/openlawlibrary/taf/commit/9d858334f9d90512012cccc4656638453bed5442

**Pull request closed without merging**